contrib/enum_metrics/enum_metrics/errant_enums.py

```diff
@@ -0,0 +1,191 @@
+import sys
+import argparse
+import json
+import dbclient as db
+import esclient as es
+import config as cf
+
+LOCALHOST = 'localhost'
+QE01 = 'qe01'
+QE02 = 'qe02'
+STAGING = 'staging'
+PROD = 'prod'
+
+
+def parse_arguments(args):
+    """Parses the supplied arguments"""
+
+    parser = argparse.ArgumentParser(prog="errant_enums.py", description='Script to delete errant enums')
+
+    subparsers = parser.add_subparsers(help='commands')
+
+    # A list command to list all excess enums.
+    list_parser = subparsers.add_parser('list', help='List all excess enums')
+
+    # A delete command to delete a given excess enum.
+    delete_parser = subparsers.add_parser('delete', help='Delete errant enum')
+
+    delete_parser.add_argument('--dryrun',
+                               action='store_true', help='Display errant enums related data for the given metric name.')
+    delete_parser.add_argument('-m', '--metricName',
+                               required=True, help='metric name to be deleted')
+    delete_parser.add_argument('-t', '--tenantId',
+                               required=True, help='tenantId corresponding to the metric name to be deleted')
+
+    environments = [LOCALHOST, QE01, QE02, STAGING, PROD]
```

Comment:
Only one of these environments is present in the `config.ini` file. It would be better to use the 'SafeConfigParser.sections()` method to get the list of available environments, instead of hard-coding them. Also, these choices are rax-specific.